src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/prop-types */
+import './book.css';
+
+import React from 'react';
+
+const Book = (props) => {
+  const { title, name } = props;
+  return (
+    <div className="book-container">
+      <p>
+        {title}
+        <span> by </span>
+        {name}
+      </p>
+      <button type="button">button</button>
```

Comment:
- [OPTIONAL] Please change the content of this button to be `remove`  instead of `button`.